xml/ay_custom_user_scripts.xml

```diff
@@ -185,11 +185,21 @@
      <emphasis>init-scripts</emphasis>.
     </para>
     <para>
-     The following elements must be between the
-     &lt;scripts&gt;&lt;init-scripts
-     config:type="list"&gt;&lt;script&gt; ...
-     &lt;/script&gt;&lt;/init-scripts&gt;...&lt;/scripts&gt;
-     tags
+     Init scripts elements must be placed as follows:
+    </para>
+<screen>&lt;scripts&gt;
+    &lt;init-scripts config:type="list"&gt;
+      &lt;script&gt;
+        ...
+      &lt;/script&gt;
+    &lt;/init-scripts&gt;
+&lt;/scripts&gt;</screen>
+
+    <para>
+     Init scripts are different from the rest of script types because they are
+     not executed by &yast; but after &yast; has finished. For this reason,
+     their XML representation is different from the used by the other script
```

Comment:
'from the used` I think there is a word missing?

Reply:
I meant that their representation is different from the representation of other script types. But I was unsure about how to express that without repeating the word *representation*. What about just "their XML representation is different from other script types"?

Reply:
That works! Thanks!